### PR TITLE
#41 extend cv parser response properties 

### PR DESCRIPTION
**Pull request by kopfrechner**

### CODE REVIEW

File: eRecruiter.Api/Responses/CvParser/CvParserResponse.cs [hunk 1]
```diff
@@ -76,5 +77,55 @@ public class CvParserResponse
...
+        /// </summary>
+        public string Summary { get; set; }
+        
+        /// <summary>
+        /// Competences of the person.
+        /// </summary>
+        public IEnumerable<string> Competences { get; set; }
+
+        /// <summary>
+        /// Interests of the persion.
```
Comment (bwolkerstorfer) [inline eRecruiter.Api/Responses/CvParser/CvParserResponse.cs]: fix typo

File: eRecruiter.Api/Responses/CvParser/EducationResponse.cs [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        /// </summary>
+        public DateTime? StartDate { get; set; }
+
+        /// <summary>
+        /// Education end date
+        /// </summary>
+        public DateTime? EndDate { get; set; }
+
+        /// <summary>
+        /// Is currently experiencing this Education
```
Comment (bwolkerstorfer) [inline eRecruiter.Api/Responses/CvParser/EducationResponse.cs]: ``is currently studying this field of education`` may be better description?

File: eRecruiter.Api/Responses/CvParser/ExperienceResponse.cs [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+{
+    /// <summary>
+    /// Description of an experience
+    /// </summary>
+    public class ExperienceResponse
+    {
+        /// <summary>
+        /// Experience start date
+        /// </summary>
+        public DateTime? StartDate { get; set; }
```
Comment (bwolkerstorfer) [inline eRecruiter.Api/Responses/CvParser/ExperienceResponse.cs]: ``ApplicantExperienceParameter`` uses ``FlexDate`` for this field

File: eRecruiter.Api/Responses/CvParser/ExperienceResponse.cs [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+        /// <summary>
+        /// Experience end date
+        /// </summary>
+        public DateTime? EndDate { get; set; }
```
Comment (bwolkerstorfer) [inline eRecruiter.Api/Responses/CvParser/ExperienceResponse.cs]: ``ApplicantExperienceParameter`` uses ``FlexDate`` for this field

File: eRecruiter.Api/Responses/CvParser/ExperienceResponse.cs [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+        /// <summary>
+        /// Is current experience
```
Comment (bwolkerstorfer) [inline eRecruiter.Api/Responses/CvParser/ExperienceResponse.cs]: ``is current workplace`` may be better description

**Comment by bwolkerstorfer** [comment]
The PR is missing update to the assembly version and the change log needs to be updated as well.

**Comment by kopfrechner** [comment]
@bwolkerstorfer applied your recommendations

**Comment by bwolkerstorfer** [comment]
change log still requires update and version needs to be adjusted

**Pull request merged**